src/tests/users/hospital/updateHospitalUser.test.js

```diff
@@ -117,10 +117,12 @@ const testCases = [
 
 const testTable = testCases.map(testCase => [testCase.title, testCase.request, testCase.response]);
 
-const hospitalUserEndpoint = '/api/hospital';
-
 test.each(testTable)('Update Hospital User endpoint: %s', (title, reqData, resData) => {
   const testGlobals = JSON.parse(process.env.TEST_GLOBALS);
+  const hospitalUserEndpoint = `/api/hospital/${testGlobals.hospitalUser.id}`;
+  // eslint-disable-next-line no-console
+  console.log(hospitalUserEndpoint);
```

Comment:
I don't think we need this console here.

Reply:
why the console is not needed?? It should be jettisoned or use the custom logger?

Reply:
I believe the log was used just for debugging purposes while implementing this task. Also, the information logged on its own is not really useful and would be too redundant given that it would be logged for each test case.

Reply:
Fixed